abis/pile.json

```diff
@@ -0,0 +1,321 @@
+[
```

Comment:
Can you remove all parts of the abi that are not used by the subquery? That cleans up the bundle quite a bit :)